.github/workflows/linters.yml

```
@@ -1,24 +1,73 @@
-name: Code Linters
+name: OpenAPI Spectral Linter
 
 on:
   pull_request:
     branches:
         - main
+
 jobs:
   spectral:
     runs-on: ubuntu-latest
-        
+    container:
+      image: node:14-alpine3.12
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Setup Node.js
-        uses: actions/setup-node@v3
-        with:
-            node-version: 14 
-
       - name: Install Spectral
         run: npm install @stoplight/spectral-cli
 
-      - name: Run Spectral Lint
-        run: npx spectral lint .
+      - name: Run Spectral Lint on Credentials API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-credentials.yaml credentials.yaml --display-only-failures || exit 1
+        
+      - name: Run Spectral Lint on Data Streaming API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-datastreaming.yaml data_streaming.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Digital Certificates API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-digital_certificates.yaml digital_certificates.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Domains API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-domains.yaml domains.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Edge Applications API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-edgeapplications.yaml edgeapplications.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Edge Firewall API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-edgefirewall.yaml edgefirewall.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Edge Functions API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-edgefunctions.yaml edgefunctions.yaml --display-only-failures || exit 1
+
+      - name: Run Spectral Lint on Edge Functions Instance Edge Firewall API
+        run: npx spectral lint --verbose --ruleset spectral/.spectral-edgefuncinstedgefirewall.yaml edgefunctionsinstance_edgefirewall.yaml --display-only-failures || exit 1
```

I know it's long, but can we write it in full length?
Don't forget to rename the file itself
```suggestion
        run: npx spectral lint --verbose --ruleset spectral/.spectral-edgefunctionsinstance_edgefirewall.yaml edgefunctionsinstance_edgefirewall.yaml --display-only-failures || exit 1
```